### PR TITLE
RDF type stored in document.

### LINKED ISSUE #74: Index rdf:type for Resources
In order to complete the Sinopia Search tickets starting with https://github.com/LD4P/sinopia_editor/issues/1412, we need to index one or more `rdf:type` for each resource.

Blocks https://github.com/LD4P/sinopia_editor/issues/1412

### PR DESCRIPTION
**Pull request by jermnelson**
Stores the RDF Type URI in an array. 

Fixes #74

### CODE REVIEW

File: src/Indexer.js [hunk 1]
```diff
@@ -246,6 +247,13 @@ export default class Indexer {
     return dates.length > 0 ? dates[0] : undefined
   }
 
+  buildRDFTypes(indexObject, json) {
+    const rdfTypes = json['@graph']
+      .filter((item) => item['@type'] && !Array.isArray(item['@type']))
```
Comment (jermnelson) [inline src/Indexer.js]: This enforces that the `@type` is a single URI and not an array.

File: config/default.js [hunk 2]
```diff
@@ -31,6 +31,10 @@ module.exports = {
         path: '$..subtitle',
         autosuggest: true
       },
+      type:  {
+        type: 'text',
```
Comment (justinlittman) [inline config/default.js]: Don't we want it to be a keyword?

File: src/Indexer.js (hunk 1 above)
Comment (justinlittman) [reply src/Indexer.js]: This doesn't seem like a great way of identifying the correct `@type`.

File: config/default.js (hunk 2 above)
Comment (jermnelson) [reply config/default.js]: Yes, changing now.

**Comment by jermnelson** [comment]
@justinlittman lastes commit changes filtering out `prov:Activity` types and supports both single URI and an array for `@type` field. Tracking down now why `indexSpy` is not being invoked in a new test.

File: __tests__/Indexer.test.js [hunk 3]
```diff
@@ -178,6 +181,41 @@ describe('Indexer', () => {
...
 
+    describe('when document has multiple types', () => {
+      it('type has multiple URIs', () => {
+        const multipleTypesJSON = {...json}
+        multipleTypesJSON['@graph'][0]['@type'] = [
+          multipleTypesJSON['@graph'][0]['@type'],
+          'http://id.loc.gov/ontologies/bibframe/WorkTitle'
+        ]
+        indexer.index(multipleTypesJSON, objectUri, objectTypes)
+        expect(indexSpy).toHaveBeenCalledWith({
```
Comment (justinlittman) [inline __tests__/Indexer.test.js]: The spy probably needs to be clearer or recreated.

**Pull request merged**